tests/ctst/features/quotas/Quotas.feature

```diff
@@ -96,3 +96,43 @@ Feature: Quota Management for APIs
             | 100        | 200         | 0            | IAM_USER |
             | 100        | 0           | 200          | IAM_USER |
             | 100        | 200         | 200          | IAM_USER |
+
+    @2.6.0
+    @PreMerge
+    @Quotas
+    @CronJob
+    @DataDeletion
+    @NonVersioned
+    Scenario Outline: Quotas are affected by deletion operations between count items runs
+        Given an action "DeleteObject"
+        And a permission to perform the "PutObject" action
+        And a STORAGE_MANAGER type
+        And a bucket quota set to 10000 B
+        And an account quota set to 10000 B
```

Comment:
what is the point, we override on next lines?

Reply:
Increasing the inflights, if no quota is set, they won't increase, and in this test, we don't run count items

Reply:
at line 121, it seems we wait for count items:
> When the "count-items" cronjobs completes without error

what i don't understand is really the "setup" part at the beginning:
* And an account quota set to 10000 B
* And an upload size of 1000 B for the object "obj-1"
* And an account quota set to <accountQuota> B

→ setting the second quota only would do the same, nothing happens in between...

Reply:
- quota of 10000
- send 1000 (inflight at 1000 afetr x seconds)
- set quota as 1000
- try to put more, should be denied because of quota
- run count items, it should update the "current" and set inflight to 0 (so, 1000 quota, 1000 current, 0 inflight)
- delete object (-1000 inflight after X seconds)
- should now be able to put an object

Reply:
I updated to have 1000 at the beginning, not a too large value...

Reply:
Oh, do you mean `an upload size of 1000 B for the object "obj-1"` is actually the same as `I PUT an object with size <uploadSize>` ?
written like this, it looks like it just sets a parameter (and what is the point of having a duplicated step?)

Also, the huge sequence of "And" makes it look like just prerequisitives, instead of the more readable specification of the test : Given x [...] and y --> When x1 Then y1 --> When x2 Then y2

Reply:
> quota of 10000
> send 1000 (inflight at 1000 after x seconds)
> set quota as 200
> try to put more, should be denied because of quota

can do the exact same thing with:
> set quota as 1000
> send 1000 (inflight at 1000 after x seconds)
> try to put more, should be denied because of quota

Reply:
> Oh, do you mean an upload size of 1000 B for the object "obj-1" is actually the same as I PUT an object with size <uploadSize> ?

yes

> Also, the huge sequence of "And" makes it look like just simple prerequisitives (it is even followed by policy setup)...

It is a prerequisite indeed, for this scenario

> can do the exact same thing with:

yes, but here we also (indirectly) test changing quotas, and I'm not against more checks...

Reply:
if we want to test changing quotas, best make it explicit IMHO
* otherwise somebody will eventually remove it... and reduce coverage
* because the purpose of Cucumber is to actually make an "excecutable spec", so it should be very explciit about what it does & tests : it is not just about having the actual scenario, but also about having a scenarios which "user" can relate to (as is, i don't think you could present this scenario to a PO or customer...)